Neos.Flow/Tests/Behavior/Features/Bootstrap/SecurityOperationsTrait.php

```diff
@@ -104,7 +104,7 @@ public function iHaveTheFollowingPolicies($string)
      */
     public static function cleanUpSecurity()
     {
-        if (file_exists(self::$testingPolicyPathAndFilename)) {
+        if (self::$testingPolicyPathAndFilename && file_exists(self::$testingPolicyPathAndFilename)) {
```

Comment:
nitpick: This is a "falsy" value but we prefer strict comparison according to the [CGL](https://neos.readthedocs.io/en/stable/References/CodingGuideLines/PHP.html#php-in-general)
```suggestion
        if (self::$testingPolicyPathAndFilename !== null && file_exists(self::$testingPolicyPathAndFilename)) {
```